### PR TITLE
PR | Issue-31 | Main layout implemented

### PR DESCRIPTION
**Pull request by VladislavGulik**

### CODE REVIEW

File: build/src/index.html [hunk 1]
```diff
@@ -8,6 +8,14 @@
     <title>Dark-Mockup-UI</title>
 </head>
 <body>
-    
+    <main class="main main_home-page">
+        <div class="main__container">
```
Comment (arsenykonohovwork) [inline build/src/index.html]: @VladislavGulik features below unacceptable;

**Comment by VladislavGulik** [comment]
I fix the code for this PR. Check out last commit and approve if everything is fine, please.

File: build/src/index.html [hunk 2]
```diff
@@ -8,6 +8,14 @@
     <title>Dark-Mockup-UI</title>
 </head>
 <body>
-    
+    <main class="main_home-page">
+        <div class="main__container">
+            <div class="main__chart-wave-widget"></div>
```
Comment (arsenykonohovwork) [inline build/src/index.html]: @VladislavGulik Unacceptable; Please follow style guide from README.md

File: build/src/index.html (hunk 1 above)
Comment (arsenykonohovwork) [reply build/src/index.html]:
```
.main
    .main__container.flex
        .flex__container.wrap

            // first row
            .flex__element.grid-10-3
            .flex__element.grid-10-7

            // second row
            .flex__element.grid-12-4
            .flex__element.grid-12-4
            .flex__element.grid-12-4
```

Comment (arsenykonohovwork) [reply build/src/index.html]: https://gist.github.com/arsenykonohovwork/b2b3e5f8a847f55a3135e2e8419ba06c#file-sassmeister-input-scss-L17

**Comment by VladislavGulik** [comment]
@arsenykonohovwork, i fixed main layout, please check it and let me now if something wrong.

**Pull request merged**